### ISSUE TITLE
Feature Request: Multiselect

### ISSUE THREAD

**Issue by trevoristall** [n4kz/react-native-material-dropdown#4]
(If this exist already and I missed how to do it, please close this)

**Comment by n4kz** [comment]
No, not implemented yet. I'll see what can be done for next release.

**Comment by YoucefMami** [comment]
I would be interested in the feature as well. Any plan for addressing this in the future?

**Comment by acomito** [comment]
@YoucefMami I could use a material design multi-select too. Are there other stable options at the moment? I'm poking around this but it's not material design: https://github.com/toystars/react-native-multiple-select

**Comment by YoucefMami** [comment]
@acomito Not to my knowledge. I also came across this package but I'm not sure it can be used in react-native:
https://github.com/JedWatson/react-select

**Comment by acomito** [comment]
@YoucefMami https://github.com/raynor85/react-native-material-selectize
